.github/actions/setup/action.yaml

```diff
@@ -40,7 +40,7 @@ runs:
       run: |
         # Disable full debug symbol generation to speed up CI build and keep memory down
         # "1" means line tables only, which is useful for panic tracebacks.
-        echo "RUSTFLAGS=-C debuginfo=1" >> $GITHUB_ENV
+        echo "RUSTFLAGS=-C force-frame-pointers=yes -C debuginfo=1" >> $GITHUB_ENV
```

Comment:
Please attach our discussion link here so that we know why we add this.

Reply:
Done